### PR TITLE
chore:  validate closed message

### PR DESCRIPTION
**Pull request by timarney**
# Summary | Résumé

Fixes: https://github.com/cds-snc/platform-forms-client/issues/4411

Add validation to closed message.

Screenshot 2024-10-16 at 9 00 27 AM

### CODE REVIEW

**Review by thiessenp-cds — DISMISSED** [review]
Works as expected. Thanks for remembering to add a live role for the dynamic update. LGTM! :)

File: i18n/translations/en/form-builder.json [hunk 1]
```diff
@@ -880,7 +880,10 @@
     "savedErrorMessage": "There was an error saving your changes. Try again later or contact Support.",
     "message": {
       "title": "Customize the closed form message",
-      "text1": "Tell people trying to access the form when and why it closed and where to go next."
+      "text1": "Tell people trying to access the form when and why it closed and where to go next.",
+      "errors": {
+        "translation": "Closed message must be translated in both official languages."
```
Comment (anikbrazeau) [inline i18n/translations/en/form-builder.json:885]:
```suggestion
        "translation": "Provide equivalent content in both official languages."
```

File: i18n/translations/fr/form-builder.json [hunk 2]
```diff
@@ -880,7 +880,10 @@
     "savedErrorMessage": "Une erreur s'est produite lors de l'enregistrement de vos modifications. Réessayez plus tard ou contacter l’équipe de soutien.",
     "message": {
       "title": " Personnaliser le message du formulaire fermé",
-      "text1": "Indiquer aux personnes qui tentent d'accéder au formulaire quand et pourquoi il s'est fermé et où aller ensuite."
+      "text1": "Indiquer aux personnes qui tentent d'accéder au formulaire quand et pourquoi il s'est fermé et où aller ensuite.",
+      "errors": {
+        "translation": "Closed message must be translated in both official languages. [FR]"
```
Comment (anikbrazeau) [inline i18n/translations/fr/form-builder.json:885]:
```suggestion
        "translation": "Veuillez fournir du contenu équivalent dans les deux langues officielles."
```

**Review by thiessenp-cds — APPROVED** [review]
LGTM :)

**Pull request merged**